app/views/clients/va/_tab_navigation.haml

```diff
@@ -21,6 +21,10 @@
         title: 'Enrollment Details',
         permission: can_view_enrollment_details? && show_demographic && show_expanded_dashboard,
       },
+      client_history_path(@client) => {
+        title: 'History',
+        permission: show_demographic || show_expanded_dashboard
```

Comment:
Shouldn't this only be visible if expanded consent is in place?

Reply:
The spec is saying "for the clients in their own CoC or if expanded consent has been provided" From what I understand the show_demographic should be allowing for the first part and show_expanded_dashboard allows for the second

Reply:
Great, thank you.